client/src/components/List.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Box, Text } from 'grommet'
+import { Blank } from 'grommet-icons'
+
+const Circle = ({ color, size }) => (
+  <Blank size={size} color={color}>
+    <circle cx={12} cy={12} r={12} fill="#000" />
+  </Blank>
+)
+
+const Ring = ({ color, size }) => (
+  <Blank size={size} color={color}>
+    <circle cx={12} cy={12} r={10} stroke="#000" strokeWidth="4" />
+  </Blank>
+)
+
+const listMarkers = {
+  circle: <Circle size="8px" color="brand" />,
+  ring: <Ring size="9px" color="brand" />
+}
+
+export const List = ({
+  children,
+  type = 'ul',
+  pad = 'none',
+  margin = 'none'
+}) => {
+  return (
+    <Box as={type} pad={pad} margin={margin}>
+      {children}
+    </Box>
+  )
+}
+
+const getListMarker = (stringOrComponent) => {
+  if (typeof stringOrComponent === 'string') {
+    const Marker = listMarkers[stringOrComponent]
+    return Marker || <></>
```

Comment:
I don't think we need to error anything out here, the circle and ring are going to be refactored into `/icons` when I get a chance anyway. And then I think it would make sense that all icons go through that process so we can keep track of everything. (aka no one offs) I just left that here for ease of use while build out the pages. I think they should all be imported so we can do tree shaking instead of importing all the icons to only use 1.